docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -119,6 +119,20 @@ $ oc get hypershiftdeployment -n default hypershift-demo -w
 $ oc get managedcluster <hypershiftDeployment.Spec.infraID>
 ```
 
+## Provision a hypershift hosted cluster on bare-metal
+
+Use the 'Agent' platform for HostedClusters with bare-metal worker nodes. The Agent platform uses the [Infrastructure Operator](https://github.com/openshift/assisted-service) (AKA Assisted Installer) to add worker nodes to a hosted cluster. For a primer on the Infrastructure Operator, see [here](https://github.com/openshift/assisted-service/blob/master/docs/hive-integration/kube-api-getting-started.md). In short, each bare-metal host should be booted with a Discovery Image that is provided by the Infrastructure Operator. The hosts can be booted manually or via user-provided automation, or by utilizing the [Cluster-Baremetal-Operator](https://github.com/openshift/cluster-baremetal-operator/blob/master/README.md) (CBO). Once booted, each host will run an agent process to facilitate discovering the host details and its installation. Each is represented by an Agent custom resource.
+
+When you create a HostedCluster with the Agent platform, HyperShift will install the [Agent CAPI provider](https://github.com/openshift/cluster-api-provider-agent) in the HyperShift control plane namespace.
+
+Upon scaling up a NodePool, a Machine will be created, and the CAPI provider will find a suitable Agent to match this Machine. Suitable means that the Agent is approved, is passing validations, is not currently bound (in use), and has the requirements specified on the NodePool Spec (e.g., minimum CPU/RAM, labels matching the label selector). You may monitor the installation of an Agent by checking its Status and Conditions.
+
+Upon scaling down a NodePool, Agents will be unbound from the corresponding cluster. However, you must boot them with the Discovery Image once again before reusing them.
+
+To use the Agent platform, the Infrastructure Operator and Hive Operator must first be installed. Please see [here](https://hypershift-docs.netlify.app/how-to/agent/create-agent-cluster/) for details.
```

Comment:
Is Hive Operator a must?

Reply:
Actually, in the context of ACM would Hive already be available?  We just need the ClusterDeployment CRD.

Reply:
I was asking myself the same question as well, i.e., is it "on" all the time? also, this is in the context of MCE

Reply:
@zanetworker Are you OK with this?

Reply:
I would say we can remove Hive as a requirement tbh